app/HutchManager/Models/ManageAgent.cs

```diff
@@ -0,0 +1,26 @@
+using HutchManager.Extensions;
+
+namespace HutchManager.Models;
+
+public class ManageAgent
+{
+  public int Id { get; set; }
```

Comment:
I'm not convinced this is ever needed?

- If we're creating an Agent record, we POST to `/agents` and don't have an id
- If we're editing an Agent record, we either POST or PUT to `/agents/{id}` and can get the id from the route.
```suggestion
```